features/dynamic-bundle-discounts.md

```diff
@@ -15,6 +16,20 @@ This documentation explains the process of setting up **Nosto - Dynamic Bundles*
 
 ### Nosto Dynamic Bundle template
 
+Nosto bundle templates needs the following mandatory variables (name intact) to be configured for proper functioning:
+1. discount type - Defines the type of discount offered (percent/dollar)
+2. discount value - Defines the amount/percent of discount offered on products
+3. secured - A flag that can be used to specify if the merchant willing to use `hash` based authentication for bundle discounts
```

Comment:
Better re-phrase it to: 
Enable `hash` based authentication for bundle discounts